src/app/shared/services/env/env.service.ts

```diff
@@ -13,7 +13,7 @@ export class EnvService {
   showAzureLoginOption: boolean = environment.showAzureLoginOption;
   natsEnvironmentName: string = environment.natsEnvironmentName;
   rootNamespace: string = environment.rootNamespace;
-
+  production: boolean = environment.production;
```

Comment:
Ohh @dawidgil good call. Do we need another way to target production (EWT chain), or think its okay to have this in staging as well?

Reply:
We can check the chain for example. Or check depending on `orgRequestEmail` property if it exists. If yes, then use email, if it is empty string, then org creator should work. It's up to you :)

Reply:
@dawidgil I think this approach as we have is good because we will want to make sure in staging that this is working as expected based on production build.